includes/functions/articles.php

```diff
@@ -260,7 +275,7 @@ function _column_title( $item, $classes = '', $data = '', $primary = false ) {
 	 */
 	function column_cb( $item ) {
 		return '<input type="checkbox" class="article-status" name="article-status[]" value="' .
-		       ( isset( $item->ID ) ? absint( $item->ID ) : '' ) . '" />';
+				( isset( $item->ID ) ? absint( $item->ID ) : '' ) . '" />';
```

Comment:
Here, line 292, 298 and 303, I get these are automated PHPCS fixes but not sure we should even have extra line breaks here to begin with? I'd almost just delete the line break and keep everything on one line, to avoid having extra whitespace show in these inputs

Reply:
Updated